reporter/src/funTest/kotlin/reporters/ExcelReporterTest.kt

```diff
@@ -38,6 +39,20 @@ class ExcelReporterTest : WordSpec({
             val outputDir = createTempDir().apply { deleteOnExit() }
             ExcelReporter().generateReport(ortResult, DefaultResolutionProvider(), CopyrightGarbage(), outputDir)
             outputDir.resolve("scan-report.xlsx").isFile shouldBe true
+
+            val actualFile = outputDir.resolve("scan-report.xlsx")
+            val actualXlsxUnpacked = createTempDir().apply { deleteOnExit() }
+            actualFile.unpackZip(actualXlsxUnpacked)
+
+            val expectedFile = File("src/funTest/assets/file-counter-expected-scan-report.xlsx")
+            val expactedXlsxUnpacked = createTempDir().apply { deleteOnExit() }
+            expectedFile.unpackZip(expactedXlsxUnpacked)
+
+            val sheet1 = "xl/worksheets/sheet1.xml"
+            val sheet2 = "xl/worksheets/sheet2.xml"
+            actualFile.isFile shouldBe true
```

Comment:
Line 53 can imo be removed, e.g. if it fails in line 54 instead you'll get needed information as well.